core/src/test/java/org/commonjava/maven/ext/core/impl/InitialGroovyManipulatorTest.java

```diff
@@ -79,7 +79,7 @@ public void shouldRemoveProjectInGroovyScript() throws Exception
 
         TestUtils.SMContainer smc = TestUtils.createSessionAndManager( userProperties );
         smc.getRequest().setPom( projectroot );
-        smc.getManager().scanAndApply( smc.getSession() );
+        smc.getManager().scanAndApply( smc.getSession() ); // FIXME this return nulls
```

Comment:
@rnc Sorry, I need to find out why this line is failing and then remove this.

Reply:
@rnc Actually, I am not sure if the current test failures are even related to this pull request.

Reply:
The tests pass on master and previous PRs so possibly needs looking at?

Reply:
Lets revert this bit please

Reply:
Done.